signing/Makefile

```diff
@@ -0,0 +1,26 @@
+include ./bbpass
```

Comment:
Oh, I just mean that for the targets in this file, it might be more robust to add single quotes around the variables that are likely to mess up the shell argument parsing. So instead of `-storepass $(KEYSTOREPASS)`, try `-storepass '$(KEYSTOREPASS)'`. This only seems to affect the lines prefixed with `@echo`, which appears to eat the double quotes around variables. To see what I mean, set `BBPASS := "with spaces"`, then run `make deploy-token` - the quotes around the password in the echoed blackberry-deploy command are gone. If you put single quotes around the argument, then the quoting is preserved.
I only noticed this because when I went to copy / paste the echoed command the missing quotes messed me up. I know there is a mention in the bbpass file about escaping special characters, but it's always nicer when the program can handle this by itself. I suppose it could also be addressed by actually running the commands instead of echoing them, which seems to preserve quoting anyway.
I am also not sure if this works the same on Windows or not. In unix shells, things in double quotes are subject to interpretation (variable substitution, etc.), but things in single quotes are treated literally. 
Not a big deal, really, just a nice-to-have when the program can handle the shell special characters for you.